docs/operations/disaster_recovery_garden.md

```diff
@@ -0,0 +1,207 @@
+
+Patch the **status subresource** of the garden resource with content similar to the snippet below, reflecting the last completed rotation steps.
+
+```yaml
+observedGeneration: 1
+credentials:
+  rotation:
+    certificateAuthorities:
+      lastInitiationFinishedTime: "2025-06-27T10:43:01Z"
+      lastInitiationTime: "2025-06-27T10:39:57Z"
+      phase: Prepared # Must be set if rotation was in Prepared phase
+    etcdEncryptionKey:
+      lastInitiationFinishedTime: "2025-06-27T10:43:01Z"
+      lastInitiationTime: "2025-06-27T10:39:57Z"
+      phase: Prepared # Must be set if rotation was in Prepared phase
+    observability:
+      lastCompletionTime: "2025-06-27T10:43:01Z"
+      lastInitiationTime: "2025-06-27T10:39:57Z"
+    serviceAccountKey:
+      lastInitiationFinishedTime: "2025-06-27T10:43:01Z"
+      lastInitiationTime: "2025-06-27T10:39:57Z"
+      phase: Prepared # Must be set if rotation was in Prepared phase
+    workloadIdentityKey:
+      lastInitiationFinishedTime: "2025-06-27T10:43:01Z"
+      lastInitiationTime: "2025-06-27T10:39:57Z"
+      phase: Prepared # Must be set if rotation was in Prepared phase
+```
+
+### Step 7: Configure Encrypted Resources
+
+* If the `garden` specifies **additional resources for encryption** in the spec, the status must be patched **before** the first reconciliation.
+* The resources defined in the following spec fields must be copied and applied to the `garden.status.encryptedResources` list:
+    * `garden.spec.virtualCluster.kubernetes.kubeAPIServer.encryptionConfig.resources`
+    * `garden.spec.virtualCluster.gardener.gardenerAPIServer.encryptionConfig.resources`
+
+### Step 8: Start Restoration
+
+**Scale Up `gardener-operator`:** Scale the `gardener-operator` deployment back up. It will now reconcile the `garden` resource with the correct initial status.
+
+### Step 9: Restoring multi-member ETCD clusters (HA configuration)
+
+* For a Garden running with an **HA configuration**, restoring **the etcd cluster requires manual steps**.
+* Follow the specific procedure documented in the `etcd-druid` repository:
+    * **Reference:** `https://github.com/gardener/etcd-druid/blob/master/docs/usage/recovering-etcd-clusters.md`
+* Apply the steps to both `etcd-main` and `etcd-events` clusters.
+
+-----
+
+## Edge Cases and Special Considerations
+
+#### 1\. Restoring from Credentials Rotation Phase `Prepared`
+
+When restoring from the `Prepared` phase, the secrets deployed in Step 1 will have two suffixes: the content hash and a second suffix indicating a non-empty `last-rotation-initiation-time` label.
+
+#### 2\. Restoring from Credentials Rotation Phase `Preparing`
+
+If the cluster is being restored from the `Preparing` phase (meaning rotation was actively in progress), you must verify the content of the secrets and the encryption state in **etcd** to ensure consistency.
```

Comment:
I do not understand these sections. What should I take away? It does not feel tangible.